public/src/main/java/com/revenuecat/purchases/CustomerInfo.kt

```diff
@@ -130,6 +131,10 @@ data class CustomerInfo constructor(
         return entitlements.all[entitlement]?.latestPurchaseDate
     }
 
+    @IgnoredOnParcel
+    override val rawData: JSONObject
+        get() = jsonObject
```

Comment:
We are already exposing `jsonObject`, maybe we should deprecate it in favor of `rawData` and make it private in a future release.

Reply:
I'll do that!

Reply:
Doe. Let me know if what I did is correct.